src/main/java/server/HTTPServer.java

```diff
@@ -27,11 +32,21 @@ private String getPath(ArrayList<String> request) {
         return request.get(0).split("\\s+")[1];
     }
 
-    private void writeResponse(String path, ReadableWriteable readerWriter) throws IOException {
-        if (paths.contains(path)) {
-            readerWriter.writeLine("HTTP/1.1 200 OK\r\nContent-Length:0\r\n");
-        } else {
-            readerWriter.writeLine("HTTP/1.1 404 Not Found\r\nContent-Length:0\r\n");
+    private Response route(String path) {
+        switch (path) {
+            case "/simple_get_with_body" -> {
+                return new Response(this.protocol, "200 OK", "Content-Length:11", "Hello world");
```

Comment:
**suggestion (non-blocking):** separate out the status code from the status text

You may already have a plan for this (and it certainly doesn't have to be in this PR) but the status text like "OK" and "Not Found" can be derived from the status code, making the API easier to use and less susceptible to user error.

Reply:
Great suggestion, and good point about the API's ease of use. I'm planning on making a lookup table that matches routes and handlers in a future PR, and I'll incorporate it then.